### PR TITLE
[docs]add TextArea section

### PR DESCRIPTION
**Pull request by unixchad**

### CODE REVIEW

File: docs/pages/TextArea.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# TextArea
+
+The `TextArea` component is a styled `<textarea>` element.
+It accepts all native HTML attributes for the element.
```
Comment (BeniCheni) [inline docs/pages/TextArea.md]: Nit: @unixchad, think this sentence could be removed that all React components are implied to accept HTML attributes, unless there's anything specific about `TextArea`, then this sentence could highlight it.

```suggestion
```

Comment (unixchad) [reply docs/pages/TextArea.md]: done

**Comment by BeniCheni** [comment]
@unixchad, sorry about the previous outdated snapshots in master that may cause the failed CI in this PR. The fix (i.e #627) landed. Would you mind rebasing for green before moving forward?

**Comment by unixchad** [comment]
rebased on top of master

**Pull request merged**